products-built-on-oxen/lokinet/guides/lokinet-on-a-soho-router.md

```diff
@@ -0,0 +1,325 @@
+# running lokinet client on a soho router
+
+the nicest way to run lokinet is on a soho router, it will make the defaults on the local network it serves use lokinet dns and even tunnel all traffic over lokinet in a way that it cannot leak.
+
+## requirements
+
+hardware with:
+
+* 1 wired interface 
+* 1 wireless interface
+* ability to run debian bookworm or later
+
+### network topology
+
+the network toplogy of this box will be the following:
+
+* the upstream network going wherever it goes to get internet on `eth0`
+* LAN range is 10.200.0.1/24 on `wifi0`
+* Lokinet range is 10.100.0.1/16 on `lokinet0`
+
+note the interface names for `eth0` and `wifi0` will very likely be different, these are placeholder values for the purposes of this tutorial, please replace them with the appropriate values on your system.
+
+### system component layout
+
+the layout of the system has the following components:
+
+* dnsmasq handles dhcp and forwards dns from LAN to lokinet and unbound.
+* lokinet handles DNS for lokinet.
+* unbound provides encrypted dns tunnel for ICANN.
+* hostapd handles wifi access point.
```

Comment:
Do you know what the modern device compability of hostapd is?  Back in the day it only supported a few specific drivers; I believe it now supports more via the mac80211 driver subsystem, but still apparently has device limitations (i.e. some hardware is simply unable to run as an access point).